Fastlane/deployment_lanes.rb

```diff
@@ -486,9 +485,9 @@
   organisation = origin_name[0]
   repository = origin_name[1]
 
-  # We set the page size to the max of 100 releaser per page so that as a quick way
-  # of avoiding pagination. This gives us more than enough release candidates for weekly or
-  # a bi-weekly build train. This lane will fail when there were more then 99 pre-releases published since the
+  # We set the page size to the max of 100 releaser per page so that as a quick way 
+  # of avoiding pagination. This gives us more than enough release candidates for weekly or 
+  # a bi-weekly build train. This lane will fail when there were more then 99 pre-releases published since the 
```

Comment:
```suggestion
  # We set the page size to the max of 100 releaser per page so that as a quick way
  # of avoiding pagination. This gives us more than enough release candidates for weekly or
  # a bi-weekly build train. This lane will fail when there were more then 99 pre-releases published since the
```